FrameworkSystem/private/standardLogging/gLoggingRoot.py

```diff
@@ -0,0 +1,132 @@
+"""
+gLogging Root
+"""
+
+__RCSID__ = "$Id$"
+
+import logging
+import time
+import sys
+
+from DIRAC.FrameworkSystem.private.standardLogging.LogLevels import LogLevels
+from DIRAC.FrameworkSystem.private.standardLogging.gLogging import gLogging
+
+
+class gLoggingRoot(gLogging):
+  """
+  gLoggingRoot is a gLogging object and it is particular because it is the first parent of the chaine.
+  In this context, it has more possibilities because it is the one that initializes the logger of the 
+  standard logging library and it configures it with the cfg file.
+
+  There is a difference between the parent gLogging and the other because the parent defines the behaviour
+  of all the gLogging objects, so it needs a specific class.  
+
+  gLoggingRoot has to be unique, because we want one and only one parent on the top of the chain: that is why 
+  we created a singleton to keep it unique. 
+  """
+
+  __configuredLogging = False
+  __instance = None
+
+  def __new__(cls):
+    """
+    Initialization of the singleton to keep gLoggingRoot unique.
+    """
+    if gLoggingRoot.__instance is None:
+      gLoggingRoot.__instance = object.__new__(cls)
+    return gLoggingRoot.__instance
+
+  def __init__(self):
+    """
+    Initialization of the gLoggingRoot object.
+    gLoggingRoot :
+    - initialize the UTC time
+    - set the correct level defines by the user, or the default
+    - add the custom level to logging: verbose, notice, always
+    - register a default backend: stdout : all messages will be displayed here
+    - update the format according to the command line argument 
+    """
+    super(gLoggingRoot, self).__init__()
+    self._logger = logging.getLogger('')
+    # this level is not the gLogging level, it is only used to send all log messages to the central logging system
+    self._logger.setLevel(LogLevels.getLevelValue('DEBUG'))
+
+    # initialization of the UTC time
+    logging.Formatter.converter = time.gmtime
```

Comment:
According to the documentation of 'time': 
"Convert a time expressed in seconds since the epoch to a struct_time in UTC in which the dst flag is always zero. "
https://docs.python.org/2/library/time.html
So it is UTC, isn't it ?